pygmt/tests/test_geopandas.py

```diff
@@ -131,3 +132,54 @@ def test_geopandas_plot3d_non_default_circle():
         style="c0.2c",
     )
     return fig
+
+
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        "int32",
+        "int64",
+        # Enable Int32/Int64 dtypes when geopandas>=0.13.3 is released with
+        # patch https://github.com/geopandas/geopandas/pull/2950
+        # pd.Int32Dtype(),
+        # pd.Int64Dtype(),
```

Comment:
> E           TypeError: Cannot interpret 'Int32Dtype()' as a data type

Disabling testing of these dtypes for now, until geopandas>=0.13.3 is out with the patch from https://github.com/geopandas/geopandas/pull/2950.